core/jobs_registry.py

```diff
@@ -203,7 +204,9 @@
     prod_validation_jobs_one_off.UserBulkEmailsModelAuditOneOffJob,
     prod_validation_jobs_one_off.UserSkillMasteryModelAuditOneOffJob,
     prod_validation_jobs_one_off.UserContributionScoringModelAuditOneOffJob,
+    prod_validation_jobs_one_off.UserCommunityRightsModelAuditOneOffJob,
```

Comment:
I think you have some merge errors...

Please order all these alphabetically, double-check that there are no duplicates.